Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMHTMLSelectElement.cpp

```diff
@@ -440,10 +440,7 @@ gboolean webkit_dom_html_select_element_get_multiple(WebKitDOMHTMLSelectElement*
 
 void webkit_dom_html_select_element_set_multiple(WebKitDOMHTMLSelectElement* self, gboolean value)
 {
-    WebCore::JSMainThreadNullState state;
-    g_return_if_fail(WEBKIT_DOM_IS_HTML_SELECT_ELEMENT(self));
-    WebCore::HTMLSelectElement* item = WebKit::core(self);
-    item->setMultiple(value);
```

Comment:
We should also preserve the functionality of this method by calling `setBooleanAttribute`.

Reply:
`item->setBooleanAttribute(multipleAttr, newMultiple);`? Will do it once I confirm that the test cases don't need any further rebaselining. :-) Will aim to get it over the line tonight.

Reply:
`item->setBooleanAttribute(WebCore::HTMLNames::multipleAttr, value);` in this case.

Reply:
I pushed wrong bit. :-( Fixing it now.

Reply:
Fixed in next push.